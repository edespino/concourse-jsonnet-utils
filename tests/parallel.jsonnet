local concourse = import 'concourse.libsonnet';
{
  parallel: concourse.Parallel([
    concourse.Get('git'),
    concourse.Get('dev-container'),
  ])
}
