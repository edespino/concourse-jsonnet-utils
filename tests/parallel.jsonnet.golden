{
   "parallel": {
      "in_parallel": [
         {
            "get": "git",
            "passed": [ ],
            "trigger": true
         },
         {
            "get": "dev-container",
            "passed": [ ],
            "trigger": true
         }
      ]
   }
}
